packs/data/ancestryfeatures.db/keen-eyes.json

```diff
@@ -27,7 +27,7 @@
         "rules": [
             {
                 "key": "FlatModifier",
-                "label": "Keen Eyes (Seek hidden or undetected within 30 feet)",
+                "label": "PF2E.SpecificRule.Halfling.KeenEyes.Label",
```

Comment:
```suggestion
```
Don't need a label for the modifier, just the toggle I think. This could maybe use a toggle? Don't know though but that's a later problem I guess.